baselines/marl_benchmark/common.py

```diff
@@ -346,7 +346,7 @@ def cal_ego_lane_dist_and_speed(env_obs: Observation, observe_lane_num):
             ).dot(dist_wp_vehicle_vector)
 
             dist_to_vehicle = lane_dist + np.sign(direction_vector) * (
-                np.linalg.norm(vec_2d(nearest_wp.pos) - vec_2d(v.position))
+                np.linalg.norm(vec_2d(nearest_wp.position) - vec_2d(v.position))
```

Comment:
Yeah, actually this is a mistake on my part.  At some point I changed the `Waypoint` class to contain a `Pose` instead of `pos` and `heading`, but this had too many ripple effects, so I changed it back.  However, I must have missed changing this one back (because I had made a mistake on this particular one and had it `nearest_wp.position` instead of `nearest_wp.pose.position`, so my pattern matching failed).